src/main/scala/org/zalando/react/nakadi/ReactiveNakadi.scala

```diff
@@ -86,6 +96,66 @@ class ReactiveNakadi {
     consumeWithOffsetSink(props, ReactiveNakadi.ConsumerDefaultDispatcher)
   }
 
+  def producerFlow(producerProperties: ProducerProperties)(implicit system: ActorSystem): Flow[ProducerMessage, Seq[String], NotUsed] = {
+    import org.zalando.react.nakadi.client.models.JsonOps._
+
+
+    import akka.http.scaladsl.model.MediaTypes._
+    import akka.http.scaladsl.model.headers._
+    import concurrent.duration._
+
+    implicit val materializer = ActorMaterializer()
+
+    val superPool = Http().superPool[Seq[String]]()
+
+    val uri = s"${producerProperties.serverProperties}${URI_POST_EVENTS.format(producerProperties.eventType)}"
+
+    val requestWithoutToken = HttpRequest(
+      method = HttpMethods.POST,
+      uri = Uri(uri),
+      headers = List(Accept(MediaRange(`application/json`)))
+    )
```

Comment:
There is actually already an implementation to post events [here](https://github.com/meln1k/reactive-nakadi/blob/f4c935a59d7804967c02961f7cfa93dc63e6c8c6/src/main/scala/org/zalando/react/nakadi/client/NakadiClientImpl.scala#L69) (Just FYI)